docs/vgclog_examples.md

```diff
@@ -0,0 +1,595 @@
+|------------------------|------------------------------|------------------------------|
+|GC cycle                |`<cycle-start>`, `<cycle-end>`| The start and end of a GC cycle.|
+|GC STW increment        |`<gc-start>`, `<gc-end>`      | The start and end of a GC increment that begins with a pause.
+|GC STW increment        | `<concurrent-kickoff>`       | The start of the initial GC increment of the global concurrent cycle that begins the initial mark operation
+|GC STW increment        | `<concurrent-global-final> ` | The start of the final GC increment of the global concurrent cycle that executes the final collection|
+|GC operations and sub-operations| `<gc-op>`                    | A GC operation such as mark or sweep, or a suboperation such as class unload. |
+
+**Note:** For more information about the XML structure of GC cycles, see [GC cycles](vgclog.md/#gc-cycles). For more information about GC cycle increments, see [GC increments and interleaving](vgclog.md#gc-increments-and-interleaving).
+
+The following examples use log excerpts to show how the different types of `gencon` cycle are logged.
+
+### Scavenge partial GC cycle
+
+The following example is taken from a `gencon` log. The output is broken down into sections with supporting text to explain the GC processing that is taking place.
+
+To search for a scavenge partial GC cycle, you can search for the `type` attribute value `scavenge` in `cycle-start` and `cycle-end` elements, or search for the `<af>` element that logs the allocation failure trigger.
+
+By default, the `gencon` partial GC cycle runs by using a single *stop-the-world* (STW) pause. The cycle performs only 1 operation,  a *scavenge* operation, which runs only on the *nursery* area. The cycle consists of a single GC increment, which is labelled using the elements shown in the following table:
+
+|GC Operation | GC increment | STW or concurrent| XML element of GC increment          | Details                                                                   |
+|-------------|--------------|------------------|--------------------------------------|---------------------------------------------------------------------------|
+|Scavenge     |Single        | STW              | `<gc-start>`, `<gc-end>` |Contains detailed information about objects being copied and weak roots processing|
+
+The scavenge partial GC cycle follows a general structure in the verbose GC log as shown. The lines are indented to help illustrate the flow and some elements are omitted for clarity:
+
+```xml
+
+<exclusive-start/>          (STW Pause starts)
+
+<af-start/>                 (allocation failure trigger recorded)
+
+<cycle-start/>              (scavenge cycle starts)
+
+<gc-start>                  (scavenge cycle increment starts)
+
+<mem-info>                  (memory status before operation)
+
+<mem></mem>                 (status of different types of memory)
```

Comment:
I know this element has attribute type', but you can also say 'areas of heap'.

Reply:
'status of different types of memory' replaced with 'status of different areas of heap' for gencon. 

WIll also make a similar change to balanced content in relevant PR:  'status of different regions of heap'.